### PR TITLE
Prison Station 2: Return to Prison Station

### PR DESCRIPTION
**Pull request by Alterist**
Fixes hole to space
Deletes a double wall somewhere
Fixes dz2 landing lights
Deletes extra iconstates
Deletes Pixel_y/x = 0s
Fixes misplaced light
slight area adjustment and rename

### CODE REVIEW

File: code/game/area/Prison_Station_FOP.dm [hunk 1]
```diff
@@ -432,7 +432,7 @@
 	icon_state = "hangar_alpha"
 
 /area/prison/hanger/research
-	name = "\improper Research Hanger"
+	name = "\improper Civilian Hanger"
```
Comment (Rohesie) [inline code/game/area/Prison_Station_FOP.dm]: What does the civilian hang with?

**Comment by Alterist** [comment]
fixed, also goddamn it cm get your grammer and spelling together

**Comment by Rohesie** [comment]
Build failed.

**Comment by Alterist** [comment]
damn it travis

**Pull request merged**